cache-core/src/main/java/io/micronaut/cache/interceptor/CacheInterceptor.java

```diff
@@ -118,7 +122,7 @@ public int getOrder() {
     @Override
     public Object intercept(MethodInvocationContext<Object, Object> context) {
         if (context.hasStereotype(CacheAnnotation.class)) {
-            InterceptedMethod interceptedMethod = InterceptedMethod.of(context);
+            InterceptedMethod interceptedMethod = InterceptedMethod.of(context, conversionService);
```

Comment:
```suggestion
            InterceptedMethod interceptedMethod = InterceptedMethod.of(context, beanContext.getConversionService());
```

Then you can get rid of the conversion service field, constructor param, and import